### ISSUE TITLE
"edit on github" links broken

### ISSUE THREAD

**Issue by Cimbali** [Cimbali/pympress#24]
The file extensions seem to be missing, see top right corner of any page of [pympress.xyz](http://pympress.xyz).

**Comment by FriedrichFroebel** [comment]
How are these created? When trying to create this on my Ubuntu device using `make clean && make update` from inside the docs folder, the links seem to be correct (with the index page linking to https://github.com/Cimbali/pympress/blob/master/docs/source/index.rst for example).

There is only one warning (which does not seem to be connected to the problem):

> copying static files... WARNING: html_static_path entry u'/home/me/test/pympress/docs/source/_static' does not exist

**Comment by Cimbali** [comment]
That's how links they are created. The warning is normal, it complains about a missing empty directory.

The links are created from the `breadcrumbs.html` file from the `sphinx_rtd_theme`, modified by the options in our [`docs/source/_template/breadcrumbs.html`](https://github.com/Cimbali/pympress/blob/master/docs/source/_template/breadcrumbs.html)

In particular, these are the interesting lines from the theme file (in the outdated v0.1.8 version that I seemed to have):

      <li class="wy-breadcrumbs-aside">
        {% if pagename != "search" %}
          {% if display_github %}
            <a href="https://{{ github_host|default("github.com") }}/{{ github_user }}/{{ github_repo }}/blob/{{ github_version }}{{ conf_py_path }}{{ pagename }}{{ source_suffix }}" class="fa fa-github"> Edit on GitHub</a>
          {% elif %}

I've now updated top the most recent version 0.2.4 and it works fine, as you describe. Thanks for prompting me to look into this, I'll close the bug now.
